term/src/terminalstate.rs

```diff
@@ -103,6 +103,7 @@ struct SavedCursor {
     wrap_next: bool,
     pen: CellAttributes,
     dec_origin_mode: bool,
+    dec_line_drawing_mode : bool,
```

Comment:
```suggestion
    dec_line_drawing_mode: bool,
```

The GH actions CI is complaining about this PR because it wants it to be `cargo fmt`'d; I think it's just this line that is upsetting it